src/Entity/Storage/EdgeEntityStorageBase.php

```diff
@@ -474,4 +479,38 @@ public function resetCache(?array $ids = NULL) {
     // when it is necessary, like in loadUnchanged().
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function loadMultiple(?array $ids = NULL) {
+    // Without $ids the caller wants to load ALL entities. To speed things up,
+    // let's assume the storage does not change during a page load; IOW, load
+    // all the entities for the first time and store them in a memory cache.
+    // For all subsequent calls for the same page load, return entities from
+    // that memory cache instead (without making API calls to Apigee).
+    if ($ids === NULL) {
+      $all_cache_ids = $this->memoryCache->get($this->entityTypeId . '-all-cache-ids');
```

Comment:
```suggestion
      $all_known_entity_ids = $this->memoryCache->get($this->entityTypeId . '-all-cache-ids');
```

maybe this variable name would be more descriptive